### PR TITLE
updated tracks, cars, and car groups

### PR DESCRIPTION
**Pull request by kj5dpl**
Updated list of cars, tracks and car groups following ACC 1.8 upgrade

### CODE REVIEW

File: public/src/components/server_config/settings.vue [hunk 1]
```diff
@@ -57,8 +57,8 @@ export default {
                 {value: "FreeForAll", label: "Mode FreeForAll"},
                 {value: "GT3", label: "Mode GT3"},
                 {value: "GT4", label: "Mode GT4"},
-                {value: "Cup", label: "Mode CUP"},
-                {value: "ST", label: "Mode ST"}
+                {value: "GTC", label: "Mode GTC"},
+
```
Comment (pedrofaria) [inline public/src/components/server_config/settings.vue:61]: I am not the best in JS... but as this is the last element in the array, the line can end up with `,` ???

File: public/src/components/server_config/bopsession.vue [hunk 2]
```diff
@@ -18,50 +18,27 @@ export default {
...
+            {value: "mount_panorama", label: "Mount Panorama"},
+            {value: "nurburgring", label: "Nürburgring GP"},
+            {value: "oulton_park", label: "Outlon Park"},
+            {value: "paul_ricard", label: "Paul Ricard"},
+            {value: "silverstone", label: "Silverstone"},
+            {value: "snetterton", label: "Snetterton"},
+            {value: "spa", label: "Spa"},
+            {value: "suzuka", label: "Suzuka"},
+            {value: "zandvoort", label: "Zandvoort"},
+            {value: "zolder", label: "Zolder"},
```
Comment (pedrofaria) [inline public/src/components/server_config/bopsession.vue:38]: same question here...

File: public/src/components/server_config/settings.vue (hunk 1 above)
Comment (richardbenson) [reply public/src/components/server_config/settings.vue:61]: Safe in actual objects (like this), not safe in JSON.

**Review by pedrofaria — APPROVED** [review]

**Comment by mikeev261** [comment]
I think you're missing Monza (I just built this branch): 

image

**Comment by kj5dpl** [comment]
> 

added :-)

**Pull request merged**